docs/cog_guides/downloader.rst

```diff
@@ -0,0 +1,547 @@
+    - ``[p]cog pin defender``
+    - ``[p]cog pin outdated_cog1 outdated_cog2``
+
+**Arguments**
+
+- ``<cogs>`` The cog or cogs to pin. Must already be installed.
+
+.. _downloader-command-cog-reinstallreqs:
+
+"""""""""""""""""
+cog reinstallreqs
+"""""""""""""""""
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]cog reinstallreqs 
+
+**Description**
+
+This command will reinstall cog requirements and shared libraries for all installed cogs.
+
+Red might ask user to use this when it clears contents of lib folder
+because of change in minor version of Python.
+
+.. _downloader-command-cog-uninstall:
+
+"""""""""""""
+cog uninstall
+"""""""""""""
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]cog uninstall <cogs>
+
+**Description**
+
+Uninstall cogs.
+
+You may only uninstall cogs which were previously installed
+by Downloader.
+
+Examples:
+    - ``[p]cog uninstall 26-Cogs defender``
+    - ``[p]cog uninstall Laggrons-Dumb-Cogs say roleinvite``
+
+**Arguments**
+
+- ``<cogs>`` The cog or cogs to uninstall.
+
+.. _downloader-command-cog-unpin:
+
+"""""""""
+cog unpin
+"""""""""
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]cog unpin <cogs>
+
+**Description**
+
+Unpin cogs - this will remove update lock from cogs.
+
+Examples:
+    - ``[p]cog unpin defender``
+    - ``[p]cog unpin updated_cog1 updated_cog2``
+
+**Arguments**
+
+- ``<cogs>`` The cog or cogs to unpin. Must already be installed and pinned.
+
+.. _downloader-command-cog-update:
+
+""""""""""
+cog update
+""""""""""
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]cog update [cogs...]
+
+**Description**
+
+Update all cogs, or ones of your choosing.
+
+Examples:
+    - ``[p]cog update``
+    - ``[p]cog update defender``
+
+**Arguments**
+
+- ``<cogs>`` The cog or cogs to update. If omitted, all cogs are updated.
+
+.. _downloader-command-cog-updateallfromrepos:
+
+""""""""""""""""""""""
+cog updateallfromrepos
+""""""""""""""""""""""
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]cog updateallfromrepos <repos>
+
+**Description**
+
+Update all cogs from repos of your choosing.
+
+Examples:
+    - ``[p]cog updateallfromrepos 26-Cogs``
+    - ``[p]cog updateallfromrepos Laggrons-Dumb-Cogs 26-Cogs``
+
+**Arguments**
+
+- ``<repos>`` The repo or repos to update all cogs from.
+
+.. _downloader-command-cog-updatetoversion:
+
+"""""""""""""""""""
+cog updatetoversion
+"""""""""""""""""""
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]cog updatetoversion <repo_name> <revision> [cogs]
+
+**Description**
+
+Update all cogs, or ones of your choosing to chosen revision of one repo.
+
+Note that update doesn't mean downgrade and therefore revision
+has to be newer than the one that cog currently has. If you want to
+downgrade the cog, uninstall and install it again.
+
+Example:
+    - ``[p]cog updatetoversion Broken-Repo e798cc268e199612b1316a3d1f193da0770c7016 cog_name``
+
+**Arguments**
+
+- ``<repo_name>`` The repo or repos to update all cogs from.
+- ``<revision>`` The revision to update to.
```

Comment:
Same note about "revision" as above.

Reply:
Added a comment pointing to `cog installversion` to get an explanation of revision